src/moj/components/date-picker/_date-picker.scss

```diff
@@ -0,0 +1,285 @@
+.moj-datepicker {
+  position: relative;
+  @include govuk-font(16);
+}
+
+.moj-datepicker-dialog {
+  display: none;
+  position: absolute;
+  top: 0;
+  min-width: 280px;
+  padding: govuk-spacing(4);
+  outline: 2px solid govuk-colour('black');
+  outline-offset: -2px;
+  background-color: govuk-colour('white');
+  transition: background-color 0.2s, outline-color 0.2s;
+  z-index: 2;
+}
+
+.moj-datepicker-dialog__header {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: govuk-spacing(2);
+}
+
+.moj-datepicker-dialog__title {
+  @include govuk-font(16);
+  font-weight: bold;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.moj-datepicker-dialog__navbuttons {
+  display: flex;
+  align-items: center;
+}
+
+.moj-datepicker-calendar {
+  border-collapse: collapse;
+  margin-bottom: govuk-spacing(4);
+
+  tbody:focus-within {
+    outline: 2px solid $govuk-focus-colour;
+  }
+
+  td {
+    border: 0;
+    margin: 0;
+    outline: 0;
+    padding: 0;
+  }
+
+  th {
+    @include govuk-font(16);
+    font-weight: bold;
+    color: $govuk-text-colour;
+  }
+
+}
+
+.moj-datepicker-dialog > .govuk-button-group {
+    margin-bottom: 0;
+
+  > * {
+    margin-bottom: 0;
+  }
+}
+
+.moj-datepicker-button {
+  @include govuk-font(16);
+    background-color: transparent;
+    outline: 2px solid rgba(0, 0, 0, 0);
+    outline-offset: -2px;
+    border-width: 0;
+    color: $govuk-text-colour;
+    height: 40px;
+    margin: 0;
+    padding: 0;
+    width: 44px;
+    position: relative;
+
+    @media (forced-colors: active) {
+      // Don't show the bottom bar in forced-color modes as it blocks the outline
+      &:after {
+        display: none
+      }
+    }
+
+    &:after {
+      content: "";
+      position: absolute;
+      bottom: 0px;
+      height: 4px;
+      left: 0;
+      right: 0;
+      background-color: transparent;
+    }
+
+    &[aria-disabled="true"],
+    &[aria-disabled="true"]:hover {
+      background-color: govuk-colour('light-grey');
+      color: govuk-colour('black');
+      cursor: not-allowed;
+      text-decoration: line-through;
+    }
+
+    &:hover {
+      color: $govuk-text-colour;
+      background-color: #949494;
+      text-decoration: none;
+      -webkit-box-decoration-break: clone;
+      box-decoration-break: clone;
+      cursor: pointer;
+    }
+
+    &:focus {
+      color: govuk-colour('black');
+      background-color: govuk-colour('yellow');
+      outline-color: transparent;
+      -webkit-box-decoration-break: clone;
+      box-decoration-break: clone;
+      &:after {
+        background-color: govuk-colour('black');
+      }
+    }
+
+    &:focus:hover {
+      background-color: #949494;
+      outline-color: govuk-colour('yellow');
+      &:after {
+        background-color: transparent;
+      }
+    }
+
+    &.current:not(:focus) {
```

Comment:
Ah yes, I wrestled with this one for a bit... my logic here is that the 'current' and 'today' aren't really modifiers of the `button`,  they *are* modifiers of `moj-datepicker-calendar__day` but then the rest of the css is all based around the button so that felt confusing, so I opted for no prefix.

In hindsight this probably doesn't work, and I just need to suck up the dubious semantic s of `moj-datepicker__button--current`